### PR TITLE
#27

### PR DESCRIPTION
**Pull request by genaforvena**
Added activity to show image on the full screen.

### CODE REVIEW

**Comment by genaforvena** [comment]
@sanovikov71 @oleg-popenov

File: app/build.gradle [hunk 1]
```diff
@@ -76,23 +76,18 @@ dependencies {
```
Comment (sanovikov71) [inline app/build.gradle:76]: Maybe it is better to create several logic groups of dependencies with some comments (if you really want to change smth here at all)

File: app/src/main/java/org/imozerov/streetartview/network/FetchService.kt [hunk 2]
```diff
@@ -19,6 +20,7 @@ class FetchService : IntentService("FetchService") {
     lateinit var dataSource: DataSource
 
     override fun onHandleIntent(intent: Intent?) {
+        Log.d(TAG, "onHandleIntent($intent)")
         if (intent != null) {
```
Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/network/FetchService.kt:29]: The injection should be moved to onCreate. It is odd that a field is injected on any intent.

File: app/src/main/java/org/imozerov/streetartview/ui/explore/list/ArtListFragment.kt [hunk 3]
```diff
@@ -32,6 +41,13 @@ class ArtListFragment : Fragment(), Filterable, ArtView {
         val layoutManager = GridLayoutManager(context, 3)
         rootView.art_objects_recycler_view.layoutManager = layoutManager
         rootView.art_objects_recycler_view.adapter = adapter
+        rootView.swipe_to_refresh_layout.setOnRefreshListener { FetchService.startFetch(context) }
+
+        fetchFinishedBroadcastReceiver = object: BroadcastReceiver() {
+            override fun onReceive(p0: Context?, p1: Intent?) {
```
Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/ui/explore/list/ArtListFragment.kt]: Naming is bad

**Pull request merged**